docs/faq/README.md

```diff
@@ -12,10 +12,75 @@ In this section you can find answers to frequently asked questions regarding jav
 - [Unit testing custom events](./unit-testing-custom-events.html)
 - [Unit testing initialization error](./unit-testing-init-error.html)
 
+## Checkbox's 'checked' attribute does not match the property
+
+Many elements reflect their properties as attributes, and vice versa, like for example the `disabled` attribute on a button.
+
+```html
+<button disabled></button>
+```
+
+```js
+console.log(myButton.disabled); // true
+console.log(myButton.hasAttribute('disabled')); // true
+```
+
+If we set the property to false, it'll _reflect_ it to an attribute. (In this case, because it's a boolean attribute, it'll be omitted)
+
+```js
+myButton.disabled = false;
+console.log(myButton.hasAttribute('disabled')); // false
+```
+
+This concept is called attribute reflection.
+
+However, this is not true for the `checked` attribute on an input element of type checkbox. The `checked` property on the input element does not reflect to an attribute, and should only be relied on to set an initial state. Consider the following example:
+
+```html
+<input id="mycheck" type="checkbox" checked></input>
+```
+
+It will only set the property the first time:
+
+```js
+console.log(mycheck.checked); // true
+```
+
+Removing the `checked` attribute does not change the `checked` _property_.
+
+```js
+mycheck.removeAttribute('checked');
+console.log(mycheck.checked); // true
+```
+
+And similarly, changing the `checked` property does not change the attribute:
+
+```js
+mycheck.checked = false;
+console.log(mycheck.hasAttribute('checked')); // true
+console.log(mycheck.checked); // false
+```
+
 ## Custom elements render life cycle
 
 <iframe src="https://andyogo.github.io/custom-element-reactions-diagram/" style="width: 940px; border: none;margin-left: -100px; height: 1350px;"></iframe>
 
+## Debugging styles in adopted stylesheets with Chrome Dev Tools
+
+If your custom element leverages the performance benefits of [constructable stylesheets](https://developers.google.com/web/updates/2019/02/constructable-stylesheets), you will likely run into [this Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=946975) that makes it difficult to debug those styles in Dev Tools. In order to make those styles accessible in Dev Tools again, you can prepend the following snippet to the beginning of your development environment to remove the `adoptedStyleSheets` API and have access to those styles in the inspector. When you find yourself needing this work around, it would be beneficial to the community for you to visit the linked Chrome bug and star it to help drive the priority of a fix in the Chrome team.
+
```

Comment:
Perhaps we can mention here that lit-element uses adopted stylesheets?

Reply:
Added.